subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_resolution.adoc

```diff
@@ -248,6 +261,37 @@ If multiple incompatible Gradle versions are in play, all should be used when se
 | `modules-2`           | `files-2.1`           | `metadata-2.90`           | Gradle 6.1
 |===
 
+[[sub:shared-readonly-cache]]
+=== Sharing the dependency cache with other Gradle instances
+
+Instead of <<sub:cache_copy,copying the dependency cache into each container>>, it's possible to mount a shared, read-only folder that will act as a dependency cache for all containers.
+This cache, unlike the classical dependency cache, is accessed without locking, making it possible for multiple builds to read from the cache concurrently.
+When Gradle will look for dependencies (artifacts or metadata), it will look into the classic writable cache, but will also search into the read-only cache.
```

Comment:
```suggestion
When Gradle looks for dependencies (artifacts or metadata), it will look into the writable cache in the local Gradle home directory, but will also search the shared read-only cache if it is activated.
```